doc/manual_experimental.rst

```diff
@@ -744,6 +669,7 @@ in any required way. For example, here is how one might define the classic
 type is an instance of it:
 
 .. code-block:: nim
+
     :test: "nim c $1"
```

Comment:
@a-mr I'm not sure what to do about this. Without a blank line, GitHub doesn't show the code block, but with a blank line, it also shows the `:test:` directive.

Reply:
there should not be lines before `:test:`, because in that case it would become part of code, not option.

It seems as a bug/feature in Github RST implementation — it actually seems to be too restrictive, it deletes the block altogether since it obviously does not recognize `:test:` option (it's not present in docutils/Sphinx).

Reply:
it's not the only case of very restrictive behavior, sometimes it's worse, see e.g. [this bug](https://github.com/github/markup/issues/1446) reported by @timotheecour 

It seems that [this opensource file](https://github.com/github/markup/blob/master/lib/github/commands/rest2html) is used for generation, it's a Python script that imports `docutils`. Unfortunately I don't have enough time to check what's wrong there :-(